fdbserver/DataDistribution.actor.cpp

```diff
@@ -1310,7 +1311,7 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 	// Five steps to create each machine team, which are document in the function
 	// Reuse ReplicationPolicy selectReplicas func to select machine team
 	// return number of added machine teams
-	int addBestMachineTeams(int targetMachineTeamsToBuild) {
+	int addBestMachineTeams(int targetMachineTeamsToBuild, int remainingMachineTeamBudget) {
```

Comment:
do not pass in remainingMachineTeamBudget

Reply:
As discussed, this one will not be changed because the extra parameter will be used to decide if we should call the build team function.